RPIPs/RPIP-34.md

```diff
@@ -0,0 +1,365 @@
+* **Purpose:** To ensure that an avenue exists to remove a current GMC Administrator.
+* **Responsible Party:** GMC Members
+* **Voting Group:** GMC Members
+* **Vote Type:** Single Choice Majority (Committee Internal)
+* **Trigger(s):** At will
+* **Other Requirements:** No other requirements.
+
+### VOP-IMC-MEMBERSELECT
+This is a voted operational process (VOP) definition, see [RPIP-34](RPIP-34.md) for more information.
+
+* **Name:** Incentives Management Committee Selection
+* **Description:** An election to fill the membership slots of the IMC.
+* **Purpose:** Ensure that the pDAO has the opportunity to periodically vote on IMC membership.
+* **Responsible Party:** IMC Members > RPIP Editors
+* **Voting Group:** pDAO
+* **Vote Type:** Weighted (Snapshot)
+* **Trigger(s):** 
+  * Periodic approximately yearly (between 10-14 months).
+  * More than half of the IMC's membership has not been elected via vote.
+  * There are vacant seats on the IMC that cannot be filled without a vote.
+* **Other Requirements:** See RPIP-10 - Management Committee Selection for full requirements.
+
+### VOP-ODAO-MEMBERSUGGEST
+This is a voted operational process (VOP) definition, see [RPIP-34](RPIP-34.md) for more information.
+
+* **Name:** oDAO Membership Suggestion
+* **Description:** A vote by the pDAO to suggest the addition or removal of a node to or from the oDAO.
+* **Purpose:** To give the pDAO non-binding input into the oDAO member set.
+* **Responsible Party:** Rocket Pool Community
+* **Voting Group:** pDAO
+* **Vote Type:** Single Choice Plurality (Snapshot)
+* **Trigger(s):** At will
+* **Other Requirements:** No further requirements.
+
+
+### VOP-ODAO-MEMBERCHANGE
+This is a voted operational process (VOP) definition, see [RPIP-34](RPIP-34.md) for more information.
+
+* **Name:** oDAO Membership Change
+* **Description:** A vote by the oDAO to add or remove a node to or from the oDAO.
+* **Purpose:** To structure and record oDAO membership votes.
+* **Responsible Party:** Rocket Pool Community
+* **Voting Group:** oDAO
+* **Vote Type:** Single Choice Plurality (On-chain)
+* **Trigger(s):** 
+  * If an instance of VOP-ODAO-MEMBERSUGGEST has a positive outcome.
+  * At will
+* **Other Requirements:** No further requirements.
+
+
+### VOP-GMC-DISPUTE
+This is a voted operational process (VOP) definition, see [RPIP-34](RPIP-34.md) for more information.
+
+* **Name:** Grant Decision Appeal
+* **Description:** A vote triggered by a member of the community to appeal a grant decision by the GMC.
+* **Purpose:** To allow the pDAO to override the GMC if it disagrees with a given grant decision.
+* **Responsible Party:** Rocket Pool Community
+* **Voting Group:** pDAO
+* **Vote Type:** Single Choice Plurality (Snapshot)
+* **Trigger(s):** Grant Announcement (within 2 weeks of announcement)
+* **Other Requirements:** See RPIP-26 - Expanded GMC Administrator Role for full requirements.
+
+Optional:
+* pDAO charter changes
+* oDAO charter changes
```

Comment:
maybe the odao responsibilities bits -- the rest of both charters are currently the _most_ difficult things in RP to change, so I definitely don't think they would use the VOP process

Reply:
Yeah, more or less agree, see discussion thread.